section2-week1/exercise7/Main.cpp

```diff
@@ -11,93 +11,96 @@ using namespace std;
 
 int main()
 {
-	int value, random, answer;
-	ans = 0;
+	int value, random;
```

Comment:
Why are you declaring these variables all the way up here if they are used later?